Cargo.toml

```diff
@@ -54,3 +54,4 @@ colored = "2.1.0"
 dprint-plugin-typescript = "0.91.1"
 dprint-plugin-vue = { git = "https://github.com/c12i/dprint-plugin-vue.git", version = "0.6.0"}
 git2 = "0.19.0"
+once_cell = "1.19.0"
```

Comment:
This is now effectively implemented in the standard library. Not something that necessarily needs to be done for this PR, but worth taking a look at https://doc.rust-lang.org/std/cell/

Doesn't cover every use case that this library does though.

Reply:
Yes I am aware of `cell`, but I think it's use-case would be when dealing with interior mutability and not initializing a shared non-mutable static variable, which is what we are doing here. Kind of similar to `lazy_static`

Reply:
Cool, no problem if it isn't the right tool here. PR is good to go then!